parkkihubi/settings.py

```diff
@@ -154,6 +156,14 @@
     'DEFAULT_VERSIONING_CLASS': 'rest_framework.versioning.NamespaceVersioning',
     'ALLOWED_VERSIONS': ('v1',),
     'DEFAULT_PERMISSION_CLASSES': ('rest_framework.permissions.IsAuthenticatedOrReadOnly',),
+    'DEFAULT_AUTHENTICATION_CLASSES': ('parkings.authentication.ApiKeyAuthentication',),
+    'DEFAULT_FILTER_BACKENDS': ('django_filters.rest_framework.DjangoFilterBackend',),
     'PAGE_SIZE': 20,
     'TEST_REQUEST_DEFAULT_FORMAT': 'json',
 }
+
+
+##############
+# Parkkihubi #
+##############
+TIME_PARKINGS_EDITABLE = timedelta(minutes=2)
```

Comment:
Personally I prefer to prefix settings with their app context, so for example `PARKINGS_TIME_EDITABLE`. This makes it easier to see what context the setting relates to. Leave it up to you.